lib/routing/wrapper.dart

```diff
@@ -1,22 +1,25 @@
+// ignore_for_file: slash_for_doc_comments
+
+import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
-import 'package:lost_found_steelhacks/authentication/auth.dart';
-import 'package:lost_found_steelhacks/authentication/user.dart';
+import 'package:lost_found_steelhacks/pages/home_page.dart';
+import 'package:lost_found_steelhacks/services/auth.dart';
 import 'package:lost_found_steelhacks/pages/login_page.dart';
-import 'package:lost_found_steelhacks/pages/map_page.dart';
-import 'package:provider/provider.dart';
 
+/**
+ * Wrapper class that checks login state
+ */
 class Wrapper extends StatelessWidget {
   const Wrapper({super.key});
 
   @override
   Widget build(BuildContext context) {
-    final authService = Provider.of<AuthService>(context);
-    return StreamBuilder<MyUser?>(
-        stream: authService.user,
-        builder: (_, AsyncSnapshot<MyUser?> snapshot) {
+    return StreamBuilder<User?>(
```

Comment:
You got rid of the provider here. Just wondering why, I had provider here so we can access user info anywhere in app.

Reply:
The way I have it we still can access it anywhere. We can go back to using the Provider, but it works the same using the singleton instance pattern:
```<ServiceClassName>.instance.<whatever you need to access>``` should work from any file

Reply:
^ followup to this is there an advantage of using Provider class instead? I haven't looked into it at all